networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc

```diff
@@ -8,13 +8,16 @@ toc::[]
 
 As a cluster administrator, you can convert your IPv4 single-stack cluster to a dual-network cluster network that supports IPv4 and IPv6 address families. After converting to dual-stack networking, new and existing pods have dual-stack networking enabled.
 
-Clusters provisioned on bare metal, {ibm-power-name}, {ibm-z-name} infrastructure, {sno}, and {vmw-full} support dual-stack networking.
-
 [IMPORTANT]
 ====
 When using dual-stack networking where IPv6 is required, you cannot use IPv4-mapped IPv6 addresses, such as `::FFFF:198.51.100.1`.
 ====
 
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about platform-specific support for dual-stack networking, see xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes#nw-ovn-kubernetes-purpose_about-ovn-kubernetes[OVN-Kubernetes purpose]
```

Comment:
```suggestion
* xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes#nw-ovn-kubernetes-purpose_about-ovn-kubernetes[OVN-Kubernetes purpose]
```

Reply:
I think the additional resources are supposed to be like this now.

Reply:
This is tricky as I need to call out in some form to check the additional resources for platform-specific support. If I add a note that might add more clutter to the small assembly intro.